### PR TITLE
Fix gettext integration with tools

### PR DESCRIPTION
**Pull request by ueno**
Fixes: #357

### CODE REVIEW

File: p11-kit/p11-kit.c [hunk 1]
```diff
@@ -49,6 +49,14 @@
 #include <stdlib.h>
 #include <unistd.h>
 
+#ifdef ENABLE_NLS
+#include <libintl.h>
+#define _(x) dgettext(PACKAGE_NAME, x)
+#else
+#define _(x) (x)
+#endif
+#define N_(x) (x)
```
Comment (tomato42) [inline p11-kit/p11-kit.c:58]: what's this for?

File: trust/trust.c [hunk 2]
```diff
@@ -54,12 +54,14 @@
 #include <stdlib.h>
 #include <unistd.h>
 
+#define N_(x) (x)
```
Comment (tomato42) [inline trust/trust.c:57]: here too, why `N_`?

**Review by tomato42 — COMMENTED** [review]
I don't get the use of `N_`...

File: p11-kit/p11-kit.c (hunk 1 above)
Comment (ueno) [reply p11-kit/p11-kit.c:58]: It's a marker for the xgettext program to extract the translatable strings into POT files, but not replace the text at run time:
https://developer.gnome.org/glib/stable/glib-I18N.html#N-:CAPS

**Review by tomato42 — APPROVED** [review]
LGTM

**Comment by ueno** [comment]
Thanks for the review.

**Pull request merged**